packages/cursorless-engine/src/testUtil/extractTargetKeys.ts

```diff
@@ -18,14 +24,17 @@ export function extractTargetKeys(target: TargetDescriptor): string[] {
         ...extractPrimitiveTargetKeys(target.active),
       ];
 
-    default:
+    case "implicit":
```

Comment:
this line was how we dropped some marks.  I had added a new target type, and forgot to handle it.  Now typescript will complain if that happens again.  lesson learned: avoid `default` if possible 😅